examples/cosmos-etcd/readme.md

```diff
@@ -42,17 +42,17 @@ Your Azure Cosmos account will be provisioned with a database (EtcdDB) and a con
 
 With Azure Cosmos etcd API, benefits include:
```

Comment:
Isn't it CosmosDB? Which is distinct from Cosmos

Reply:
Cosmos etcd API is distinct from both of them. :)

But yeah, it's backed by cosmosdb under the hood